README.md

```diff
@@ -43,8 +48,8 @@ or `Triangulation_3`); however some packages serve special needs:
 
 More Information
 ================
-* [The CGAL web site](https://www.cgal.org/)
+* [The CGAL website](https://www.cgal.org/)
 * [Latest CGAL release documentation pages](https://doc.cgal.org/)
 * [Latest CGAL master documentation pages, updated once a week](https://cgal.geometryfactory.com/CGAL/doc/master/)
 * [CGAL daily testsuite results](https://cgal.geometryfactory.com/CGAL/testsuite/)
-* [Guidelines for CGAL developers](https://github.com/CGAL/cgal/wiki/Guidelines) and [Informations for new developers](https://github.com/CGAL/cgal/wiki/Information-for-New-Developers)
+* [Guidelines for CGAL developers](https://github.com/CGAL/cgal/wiki/Guidelines) and [Information for new developers](https://github.com/CGAL/cgal/wiki/Information-for-New-Developers)
```

Comment:
This modification is the only one that makes sense. I will commit it directly in `master`, and then close this PR.

Reply:
Thanks, @lrineau sir, looking forward to contribute more to CGAL. I am starting with docs now and will shift to issues and bugs as well.
I wanted to contact you regarding some issues. I tried contacting you through the email on your profile but didn't get a reply. Can you give me you official email please.